.github/workflows/test-command.yml

```diff
@@ -92,6 +92,7 @@ jobs:
         id: test
         env:
           ACTION_RUN_ID: ${{github.run_id}}
+          NORMALIZATION_TEST_TARGET: redshift
```

Comment:
Are you running the tests locally?

Is this just for testing and to be removed afterward?

Reply:
yes I will remove, I am debugging tests on CI

Reply:
I have reverted all this debug commits